pages.md

```diff
@@ -523,6 +523,13 @@
     <td>:heavy_check_mark:</td>
     <td>:heavy_check_mark:</td>
     <td>:heavy_check_mark:</td>
+  </tr><tr>
```

Comment:
Pages file is auto generated dont change this

Reply:
done revise to unchanged